test/tests/neml_simple/le_stress.i

```diff
@@ -1,28 +1,25 @@
-#
-#
-#
 [GlobalParams]
-      displacements = 'disp_x disp_y disp_z'
-[]
+  displacements = 'disp_x disp_y disp_z'
+[../]
```

Comment:
Don't add `../` to these top level blocks. That syntax used to not even work with `../` there. We're trying to go away from these nerdy `./`, `../` strings in the block open/close entirely.  Feel free to remove all of them.